uber/templates/preregistration/confirm.html

```diff
@@ -41,6 +41,10 @@
     </div>
 {% endif %}
 
```

Comment:
The way the code works is that the Javascript in ``preregbase.html`` now looks at whether or not divs exist with ids ``reg-types`` and/or ``badge-types`` and adds the form elements if they're present.